lib/posix/inotify.nim

```diff
@@ -80,9 +80,12 @@ proc inotify_rm_watch*(fd: cint; wd: cint): cint {.cdecl,
 iterator inotify_events*(evs: pointer, n: int): ptr InotifyEvent =
   ## Abstract the packed buffer interface to yield event object pointers.
   ##   ```Nim
-  ##   var evs = newSeq[byte](8192)        # Already did inotify_init+add_watch
-  ##   while (let n = read(fd, evs[0].addr, 8192); n) > 0:     # read forever
-  ##     for e in inotify_events(evs[0].addr, n): echo e[].len # echo name lens
+  ##   import std/posix  # needed for FileHandle read procedure
+  ##   const MaxWatches = 8192
+  ##   ...
+  ##   var evs = newSeq[byte](MaxWatches)  # only after inotify_init and add_watch
+  ##   while (let n = read(fd, evs[0].addr, MaxWatches); n) > 0:  # blocks until any events have been read
+  ##     echo (e[].wd, e[].mask, cast[cstring](addr e[].name)       # echo watch id, mask, and name value of each event
```

Comment:
```suggestion
  ##     echo e[].wd, e[].mask, cast[cstring](addr e[].name)       # echo watch id, mask, and name value of each event
```
Brackets are unbalanced otherwise.
Wonder if it might be worthwhile to make it a runnable example to make sure it compiles (Making sure to add a file opening and attach `-r:off`)

Reply:
Thank you.
Fixed a missing comma.
And making it a runnable example was a good idea - I've fixed a couple really stupid bugs.